lib/modules/datasource/go/base.ts

```diff
@@ -254,6 +274,7 @@ export class BaseGoDatasource {
         };
       }
       case 'azure': {
+        /* istanbul ignore next: can never happen, makes lint happy */
```

Comment:
remove if unreachable or find a test case.

maybe some old azure urls? or selfhosted devops server urls?

Reply:
readded the previous azure devops source test but changed the domain to a visualstudio.com based hostname